README.md

```diff
@@ -209,3 +209,21 @@ Environment variables
 *RAVEN_DSN*
   The DSN for Sentry
 
+
+Concourse CI
+---------------------
+
+To login and sync:
+
+> $ fly login --concourse-url https://concourse.example.com -t target  
+> $ fly -t target sync
+
+To add a pipeline:
+
+> $ fly validate-pipeline --config .ci/pipeline.yml  
+> $ fly -t target set-pipeline --config .ci/pipeline.yml --pipeline pipeline-name>--load-vars-from .ci/vars.yml  
```

Comment:
I think `--pipeline pipeline-name>--load-vars-from` should be `--pipeline <pipeline-name> --load-vars-from`.